manage_sections/views.py

```diff
@@ -164,7 +163,12 @@ def create_section_form(request):
                 'No sections found for Canvas course %s' % canvas_course_id
             )
             return render(request, 'manage_sections/error.html', status=500)
+        print('Going through canvas sections')
         for section in canvas_sections:
+            if section.get('enrollments'):
```

Comment:
This seems to duplicate the `if` below.

Reply:
That's right! thanks. The second if was put when we changed the call for the timeout for large sections.